core/compiler/src/main/scala/com/fortysevendeg/exercises/compiler/codeGen.scala

```diff
@@ -0,0 +1,142 @@
+package com.fortysevendeg.exercises
+package compiler
+
+import scala.language.implicitConversions
+import scala.reflect.api.Universe
+
+case class CodeGen[U <: Universe](override val u: U = scala.reflect.runtime.universe) extends TreeHelpers[U] {
+  import u._
+
+  def emitExercise(
+    name: Option[String]
```

Comment:
Which part of the formatting? The parameter names? I specifically had them on separate lines-- scalariform followed my lead and left them that way. If you put them back on the same line, it won't reformat them this way.